sos/plugins/watchdog.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2018 Red Hat, Inc., Reid Wahl <nwahl@redhat.com>
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+'''Collect info from watchdog to add to sosreport.
+
+Classes:
+Watchdog -- collects watchdog configs, custom executables, and logs
+'''
+
+import re
+
+from sos.plugins import Plugin, RedHatPlugin
+
```

Comment:
It's different from typical python conventions but we normally place the `sos` module imports at the top, followed by any standard library or other import statements.